app.py

```diff
@@ -627,10 +627,9 @@ def filterparsetrees():
 					n=n + 1,
 					prob=probstr(prob),
 					urlprm=urlencode(dict(urlprm, n=n + 1)),
-					tree=DrawTree(tree, senttok).text(
-						unicodelines=True, html=True, funcsep='-', morphsep='/',
-						nodeprops='t%d' % (n + 1), maxwidth=30))
```

Comment:
I notice that `gtree()` hardcodes `nodeprops='t1'` whereas here the number is variable. What is the effect of this?